### DISCUSSION TITLE
How to disable Noice for quick-fix list?

### DISCUSSION THREAD

**Discussion by danielo515** [LazyVim/LazyVim#794] [Q&A] [answered]
Hello.
I really like how noice makes some parts of neovim that are traditionally hard to deal with, like the messages window, a lot more useful.
However, for the quick-fix list window it is less useful for me to treat it like a normal buffer. I was wondering if there is a way to prevent noice from modifying the quick fix list buffer type?

Thanks in advance

**Comment by folke** [comment, marked as answer]
Noice doesn't do anything with the quickfix list? What do you mean? Can you show a screenshot?

**Reply by danielo515** [reply]
This is the part I like, when I open messages it goes into a scratch buffer:
<img width="1295" alt="image" src="https://github.com/LazyVim/LazyVim/assets/2270425/2c869ee0-2094-4606-a67a-4f8d2ffd25d7">

Something similar happens with quickfix list:
<img width="954" alt="image" src="https://github.com/LazyVim/LazyVim/assets/2270425/fb637a30-6ea9-4eba-84d4-f317416250f5">

If that is nothing lazy or noice does, then sorry for the misunderstanding.
The reason I thought it was being rendered as a normal buffer is because some keymaps that I had for other buffers were applied to the quickfix-list too, but maybe that is my mistake.

**Reply by folke** [reply]
That's just the regular quickfix list. Noice doesn't change that at all

**Reply by danielo515** [reply]
My bad then. Sorry

**Reply by hinogi** [reply]
I have the same flickering issue when the editor snack is open
